charts/backstage/README.md

```diff
@@ -0,0 +1,261 @@
+# Backstage Helm Chart
+
+![Version: 0.5.4](https://img.shields.io/badge/Version-0.5.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+A Helm chart for deploying a Backstage application
+
+---
+
+[Backstage](https://backstage.io) is an open platform for building developer portals. Powered by a centralized software catalog, Backstage restores order to your microservices and infrastructure and enables your product teams to ship high-quality code quickly — without compromising autonomy.
+
+Backstage unifies all your infrastructure tooling, services, and documentation to create a streamlined development environment from end to end.
+
+## TL;DR
+
+```console
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add backstage https://backstage.github.io/charts
+
+helm install my-release backstage/backstage
+```
+
+## Introduction
+
+This chart bootstraps a [Backstage](https://backstage.io/docs/deployment/docker) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+- [Backstage container image](https://backstage.io/docs/deployment/docker)
+
+## Usage
+
+Chart is available in the following formats:
+
+- [Chart Repository](https://helm.sh/docs/topics/chart_repository/)
+- [OCI Artifacts](https://helm.sh/docs/topics/registries/)
+
+### Installing from the Chart Repository
+
+The following command can be used to add the chart repository:
+
+```console
+helm repo add backstage https://backstage.github.io/charts
+helm install my-release backstage/backstage
+```
+
+Once the chart has been added, install one of the available charts:
+
+```console
+helm upgrade -i <release_name> backstage/backstage
+```
+
+### Installing from an OCI Registry
+
+Chart is also available in OCI format. The list of available releases can be found [here](https://github.com/backstage/charts/pkgs/container/charts%2Fbackstage).
+
+Install one of the available versions:
+
+```shell
+helm upgrade -i oci://ghcr.io/backstage/charts/backstage --version=<version>
+```
+
+> **Tip**: List all releases using `helm list`
+
+### Uninstalling the Chart
+
+To uninstall/delete the `my-backstage-release` deployment:
+
+```console
+helm delete my-backstage-release
```

Comment:
We should probably use the `helm uninstall` command here? Just because `delete` has been renamed to `uninstall`: https://helm.sh/docs/helm/helm_delete/

Reply:
Agreed, fixed. I've copy pasted it from the original readme without thinking too much about it. :speak_no_evil: